core/src/main/java/org/elasticsearch/transport/TcpTransport.java

```diff
@@ -1015,9 +1015,14 @@ protected void innerOnFailure(Exception e) {
     protected abstract Channel bind(String name, InetSocketAddress address) throws IOException;
 
     /**
-     * Closes all channels in this list
+     * Closes all channels in this list. If the synchronous boolean is set to true, the channels must be
+     * closed before the method returns. This boolean should never be called with true from a network
+     * thread.
+     *
+     * @param channels the channels to close
+     * @param synchronous whether the channels should be closed synchronously
```

Comment:
it's a matter of taste so I will leave it to you my suggestion is to call it `blocking` instead. but again up to you!